timemachine/cpp/src/bound_potential.hpp

```diff
@@ -11,12 +11,13 @@ namespace timemachine {
 // a potential bounded to a set of parameters with some shape
 struct BoundPotential {
 
-    BoundPotential(std::shared_ptr<Potential> potential, const int size, const double *h_p);
+    BoundPotential(std::shared_ptr<Potential> potential, const std::vector<double> params);
 
     int size;
     std::unique_ptr<DeviceBuffer<double>> d_p;
     std::shared_ptr<Potential> potential;
-    const int max_size_;
+
+    void set_params(const std::vector<double> params);
```

Comment:
const std::vector<double> &params (note the missing &)

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1119/commits/96a22f13e34205baea99ae5a31d769897ecb1416